lib/editor/atto/plugins/kalturamedia/yui/build/moodle-atto_kalturamedia-button/moodle-atto_kalturamedia-button-debug.js

```diff
@@ -120,7 +120,17 @@ Y.namespace('M.atto_kalturamedia').Button = Y.Base.create('button', Y.M.editor_a
         
         return courseId;
     },
-            
+
+    _replaceKafUriWithToken: function(sourceUrl, kafUri, kalturaToken) {
+        sourceUrl = this._removeProtocolFromUrl(sourceUrl);
+        kafUri = this._removeProtocolFromUrl(kafUri);
+        return sourceUrl.replace(kafUri, kalturaToken);
```

Comment:
@gonenradai - the result of this particular function indeed returns a URL without a schema, however, the schema is being added to the page later on (line 158).

This is the WYSIWYG fix, there will be another PR which fixes the way we actually save the source URL in the DB, but with the correct path.